docs/overrides/main.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block extrahead %}
```

Comment:
for my own learning, why `extrahead` instead of `script`?

Reply:
Slight preference to include async analytics in the `<head>` (via `extrahead` block) instead of the footer (via the `scripts` block) to get the script running earlier. I've typically seen analytics in the header for this reason. In practice either would work fine here